lib/datasource/hex/index.js

```diff
@@ -12,15 +12,17 @@ function getHostOpts() {
 }
 
 async function getPkgReleases({ lookupName }) {
-  const hexUrl = `https://hex.pm/api/packages/${lookupName}`;
+  let depName = '';
+  if (lookupName) depName = lookupName.split(':')[0];
```

Comment:
This logic needs to be documented, perhaps in a comment here. In which cases would the lookupName have a `:` and why then are we only interested in the first part of it?